components/post-term-list/item.tsx

```diff
@@ -1,26 +1,27 @@
 import { useContext } from '@wordpress/element';
-import PropTypes from 'prop-types';
+import { FC } from 'react';
 import { PostTermContext } from './context';
 
-export const ListItem = (props) => {
+export type ListItemProps = {
+	tagName?: keyof HTMLElementTagNameMap | undefined;
```

Comment:
Hey @fabiankaegy Why was it necessary to add `| undefined` if you have already used the optional operator (`?`) 🤔